python-sdk/src/astro/sql/operators/dataframe.py

```diff
@@ -130,7 +130,7 @@ def __init__(
             **kwargs_with_datasets(kwargs=kwargs, output_datasets=self.output_table),
         )
 
-    def execute(self, context: Context) -> Table | pd.DataFrame:
+    def execute(self, context: Context) -> BaseTable | pd.DataFrame | list:
```

Comment:
Will the list always be a `list[pd.DataFrame]`?

If so we should change it to:

```suggestion
    def execute(self, context: Context) -> BaseTable | pd.DataFrame | list[pd.Dataframe]:
```

Reply:
Not necessarily. It could be a list of anything, we're just doing our special storage method on dataframes. So [1, "foo", DataFrame, 4] would be legal here